### DISCUSSION TITLE
Recommended form validation library

### DISCUSSION THREAD

**Discussion by notramo** [svelteuidev/svelteui#351] [Q&A]
What form validation library would you recommend that works well with SvelteUI widgets?
I'm not looking for a *schema* validation library, like Zod, or Yup, but a *form* validation library, that e.g. tracks input field touched or dirty state (like svelte-forms).

**Comment by BingeCode** [comment]
Have you tried [svelte-forms-lib](https://svelte-forms-lib-sapper-docs.vercel.app/introduction)?

**Comment by notramo** [comment]
@BingeCode, I'm interested in what are the advantages of using this library with SvelteUI widgets? How does it compare to other libraries in terms of integrating with this project?

**Comment by BingeCode** [comment]
Svelte-form-libs is a library designed for form management in Svelte applications, offering seamless integration with SvelteUI widgets. It simplifies form management, provides compatibility with Svelte, enables reactive form states, custom validation, and extensibility through plugins. Compare it with other Svelte form libraries like Svelte Forms Lib and Felte based on your project needs.
I'm gonna use it in my own project and will report here when I have some experience using it ;)
